content/docs/aro/blob-storage-csi/index.md

```diff
@@ -0,0 +1,315 @@
+    apiVersion: security.openshift.io/v1
+    EOF
+
+    oc describe scc csi-azureblob-scc
+    ```
+
+1. Use helm to install the driver. 
+
+    > Note Blob Fuse Proxy is not supported for ARO yet, so we disable it.
+
+    ```bash
+    helm repo add blob-csi-driver https://raw.githubusercontent.com/kubernetes-sigs/blob-csi-driver/master/charts
+
+    helm repo update
+
+    helm install blob-csi-driver blob-csi-driver/blob-csi-driver --namespace ${CSI_BLOB_PROJECT} --set linux.distro=fedora --set node.enableBlobfuseProxy=false --set node.cloudConfigSecretNamespace=${CSI_BLOB_PROJECT} --set node.cloudConfigSecretName=${CSI_BLOB_SECRET} --set controller.cloudConfigSecretNamespace=${CSI_BLOB_PROJECT} --set controller.cloudConfigSecretName=${CSI_BLOB_SECRET}
+
+    ```
+
+## Test the CSI driver is working
+
+1. The first step for the test is the creation of the storage account and the blob container.
+
+    ```bash
+    export APP_NAME=Myapp
+
+    #bash
+    export STORAGE_ACCOUNT_NAME="stweblob""${APP_NAME,,}"
+    #zsh
+    export STORAGE_ACCOUNT_NAME="stweblob""${APP_NAME:l}"
+
+    export BLOB_CONTAINER_NAME=aroblob
+
+    az storage account create --name $STORAGE_ACCOUNT_NAME --kind StorageV2 --sku Standard_LRS --location $LOCATION -g $RG_NAME 
+
+    export AZURE_STORAGE_ACCESS_KEY=$(az storage account keys list --account-name $STORAGE_ACCOUNT_NAME -g $RG_NAME --query "[0].value" | tr -d '"')
+    
+    az storage account list -g $RG_NAME -o tsv
+
+    az storage container create --name $BLOB_CONTAINER_NAME --account-name $STORAGE_ACCOUNT_NAME
+
+    az storage container show --name $BLOB_CONTAINER_NAME --account-name $STORAGE_ACCOUNT_NAME
+
+    ```
+
+1. At this point, you must give permissions to the driver to access the Blob storage.
+
+    Go to Azure Active Directory and locate the  identity you created manually in the previous section. 
+    ![Image](images/blob-storage0.png)
+
+    After getting the name of your identity, you must give it access in the StorageAccount you created in the previous step.
+
+    ![Image](Images/blob-storage1.png)
+    ![Image](Images/blob-storage2.png)
+    ![Image](Images/blob-storage3.png)
+    ![Image](Images/blob-storage4.png)
+    ![Image](Images/blob-storage5.png)
+
+    We need to create another project where the testing pod will run. 
+    ```bash
+    export CSI_TESTING_PROJECT=testing-project
+    oc new-project ${CSI_TESTING_PROJECT}
+    ```
+
+
+1. Now, you are ready to create the storage class, the persistent volume claim and the testing pod. 
+
+    ```bash
+    cat <<EOF | oc apply -f -
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: blob
+    provisioner: blob.csi.azure.com
+    parameters:
+      resourceGroup:  $RG_NAME
+      storageAccount: $STORAGE_ACCOUNT_NAME
+      containerName:  $BLOB_CONTAINER_NAME
+      # server: SERVER_ADDRESS  # optional, provide a new address to replace default "accountname.blob.core.windows.net"
```

Comment:
don't need this, its very unlikely they'd use a different server address.

Reply:
deleted.